### PR TITLE
[zh-cn]Translate Node-v1

### PR DESCRIPTION
**Pull request by yaolixin-creater**
Tranlate content/en/docs/reference/kubernetes-api/cluster-resources/node-v1.md

#34733

### CODE REVIEW

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/node-v1.md [hunk 1]
```diff
@@ -0,0 +1,1493 @@
...
+
+<!-- 
+api_metadata:
+  apiVersion: "v1"
+  import: "k8s.io/api/core/v1"
+  kind: "Node"
+content_type: "api_reference"
+description: "Node is a worker node in Kubernetes."
+title: "Node"
+weight: 1
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/node-v1.md:20]: Missed `auto_generated: true` in original metadata.

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/node-v1.md [hunk 2]
```diff
@@ -0,0 +1,1493 @@
...
+      uid 是引用的 configMap 的 metadata.uid。
+      该字段在 Node.spec 中是禁止的，在 Node.status 中是必需的。
+
+- **externalID** (string)
+
+  <!-- 
+  Deprecated. Not all kubelets will set this field. Remove field after 1.13. see: https://issues.k8s.io/61966 
+  -->
+  已弃用。并非所有 kubelet 都会设置此字段。
+  1.13 的版本之后会删除该字段。见： https://issues.k8s.io/61966。
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/node-v1.md:162]: The ending `。` would be treat as part of URI if not spaced.
```suggestion
  1.13 的版本之后会删除该字段。见： https://issues.k8s.io/61966 。
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/node-v1.md [hunk 3]
```diff
@@ -0,0 +1,1493 @@
...
+    与污点键对应的污点值。
+
+- **unschedulable** (boolean)
+
+  <!-- 
+  Unschedulable controls node schedulability of new pods. By default, node is schedulable. More info: https://kubernetes.io/docs/concepts/nodes/node/#manual-node-administration 
+  -->
+  unschedulable 控制新 Pod 的节点可调度性。
+  默认情况下，节点是可调度的。
+  更多信息： https://kubernetes.io/docs/concepts/nodes/node/#manual-node-administration。
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/node-v1.md:248]:
```suggestion
  更多信息： https://kubernetes.io/docs/concepts/nodes/node/#manual-node-administration 。
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/node-v1.md (hunk 1 above)
Comment (yaolixin-creater) [reply content/zh-cn/docs/reference/kubernetes-api/cluster-resources/node-v1.md:20]: The sentence is meaningless and was suggested to be removed by other reviewers

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/node-v1.md [hunk 4]
```diff
@@ -0,0 +1,1493 @@
...
+    -->
+    error 描述了将 spec.configSource 与活动配置协调的任何问题。
+    可能会发生 error，例如，尝试将 spec.configSource 检查点到本地分配记录，
+    尝试检查与 spec.configSource 关联的有效负载，尝试加​​载或验证分配的配置等。
+    同步配置时可能会在不同点发生错误，较早的错误（例如下载或检查点错误）不会导致回滚到 LastKnownGood，
+    并且可能会在 Kubelet 重试后解决。
+    以后的错误（例如加载或验证检查点配置）将导致回滚到 LastKnownGood。
+    在后一种情况下，通常可以通过修复 spec.sonfigSource 中分配的配置来解决错误。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/node-v1.md:574]:
```suggestion
    在后一种情况下，通常可以通过修复 spec.sonfigSource 中 assigned 配置来解决错误。
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/node-v1.md [hunk 5]
```diff
@@ -0,0 +1,1493 @@
...
+    你可以通过在 Kubelet 日志中搜索错误消息来找到更多的调试信息。
+    error 是错误状态的人类可读描述；机器可以检查 Error 是否为空，但不应依赖跨 Kubelet 版本的 Error 文本的稳定性。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/node-v1.md:576]:
```suggestion
    error 是错误状态的人类可读描述；机器可以检查 error 是否为空，但不应依赖跨 Kubelet 版本的 error 文本的稳定性。
```

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/node-v1.md [hunk 6]
```diff
@@ -0,0 +1,1493 @@
...
+
+  - **config.lastKnownGood** (NodeConfigSource)
+    <!--
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/node-v1.md:579]: 579 前面加空行

File: content/zh-cn/docs/reference/kubernetes-api/cluster-resources/node-v1.md [hunk 7]
```diff
@@ -0,0 +1,1493 @@
...
+    以后的错误（例如加载或验证检查点配置）将导致回滚到 LastKnownGood。
+    在后一种情况下，通常可以通过修复 spec.sonfigSource 中分配的配置来解决错误。
+    你可以通过在 Kubelet 日志中搜索错误消息来找到更多的调试信息。
+    error 是错误状态的人类可读描述；机器可以检查 Error 是否为空，但不应依赖跨 Kubelet 版本的 Error 文本的稳定性。
+
+  - **config.lastKnownGood** (NodeConfigSource)
+    <!-- 
+    LastKnownGood reports the checkpointed config the node will fall back to when it encounters an error attempting to use the Assigned config. The Assigned config becomes the LastKnownGood config when the node determines that the Assigned config is stable and correct. This is currently implemented as a 10-minute soak period starting when the local record of Assigned config is updated. If the Assigned config is Active at the end of this period, it becomes the LastKnownGood. Note that if Spec.ConfigSource is reset to nil (use local defaults), the LastKnownGood is also immediately reset to nil, because the local default config is always assumed good. You should not make assumptions about the node's method of determining config stability and correctness, as this may change or become configurable in the future. 
+    -->
+    lastKnownGood 报告节点在尝试使用分配的配置时遇到错误时将回退到的检查点配置。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/cluster-resources/node-v1.md:582]: 582 前加空行

Comment (tengqm) [reply content/zh-cn/docs/reference/kubernetes-api/cluster-resources/node-v1.md:582]:
```suggestion
    lastKnownGood 报告节点在尝试使用 assigned 配置时遇到错误时将回退到的检查点配置。
```

**Comment by yaolixin-creater** [comment]
后半部分未修改，提交中，请先不要review

**Pull request merged**